.changeset/small-sides-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+Fixed #7860: The css parser, with `tailwindDirectives` enabled, will now accept `@plugin` options.
```

Comment:
Any particular reason why you didn't add the link? Remember that the changelog will go on the website too, which would make this not a link

Reply:
I just forgot :sweat_smile: